cypress/e2e/file-upload-new-UI.cy.js

```diff
@@ -0,0 +1,33 @@
+describe("tests for uploaded files shown on editor", () => {
+
+    beforeEach(() => {
+        cy.visit("/");
+    });
+
+    it("will click on editor placeholder to upload a file in WORD, and will check if there is text shown in editor ", () => {
+       
+        cy.get('[data-test="editor-placeholder-upload"]')
+            .click()
+            .selectFile("cypress/fixtures/test.docx");
+        cy.get('[data-test="editor"]').contains("lider asd").should("exist");
```

Comment:
I believe here it is ideal if we check whether the editor has exactly the uploaded text , therefore we might want to utilize has.text instead of contains, i.e. something like `cy.get('[data-test="editor"]').should("have.text", "lider asd");`. If you think this is slightly better then make this change (in the other `it` methods as well).

Either way feel free to merge this.